src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -27,7 +30,7 @@ def load_arguments(self, _):
         with self.argument_context(scope) as c:
             c.argument('account_name', completer=None)
             c.ignore('resource_group_location')
-            c.argument('locations', nargs='+', validator=validate_locations, help="space-separated locations in 'regionName=failoverPriority' format. E.g eastus=0 westus=1. Failover priority values are 0 for write regions and greater than 0 for read regions. A failover priority value must be unique and less than the total number of regions. Default: single region account in the location of the specified resource group.")
+            c.argument('locations', nargs='+', action=CreateLocation, help="space-separated key-value pairs for properties of a location formatted [KEY=VALUE ...]. Required keys are 'regionname' and 'failoverpriority'. Optional boolean key is 'iszoneredundant'. E.g. regionname=eastus failoverpriority=0 iszoneredundant=false. Failover priority values are 0 for write regions and greater than 0 for read regions. A failover priority value must be unique and less than the total number of regions. Default: single region account in the location of the specified resource group. Multiple locations can be specified by using more than one `--locations` argument.")
```

Comment:
Also, do include an example for this command and its updated usage in your `help.py` file.